src/journal/journal-verify.c

```diff
@@ -60,10 +60,11 @@ static void draw_progress(uint64_t p, usec_t *last_usec) {
 }
 
 static uint64_t scale_progress(uint64_t scale, uint64_t p, uint64_t m) {
+        /* Calculates scale * p / m, but handles m == 0 safely, and saturates.
+         * Currently all callers use m >= 1, but we keep the check to be defensive.
+         */
 
-        /* Calculates scale * p / m, but handles m == 0 safely, and saturates */
-
-        if (p >= m || m == 0)
+        if (p >= m || m == 0) /* lgtm [cpp/constant-comparison] */
```

Comment:
It seems that the alert this comment was supposed to suppress is still [there](https://lgtm.com/projects/g/systemd/systemd/alerts/?mode=tree&ruleFocus=2154840804). I'm not sure what's going on. Maybe it should start with `//` to be treated specially.